### ISSUE TITLE
Using an unsupported version of DOCKER COMPOSE with version 3.1.4

### ISSUE THREAD

**Issue by drupov** [lando/lando#3010]
Ubuntu 20.04 LTS, with the just released version `lando-v3.1.4` when I do a `lando rebuild` on my project I get a warning

```
 ■ Using an unsupported version of DOCKER COMPOSE
   You have version 1.29.1 but Lando wants something in the 1.23.0 - 1.29.0 range.
   If you have purposefully installed an unsupported version and know what you are doing
   you can probably ignore this warning. If not we recommend you use a supported version
   as this ensures we can provide the best support and stability.
   https://docs.docker.com/compose/install/#install-compose-on-linux-systems
```

I have not installed docker-compose separately, it is using the one shipped with Lando.

The docker-compose update is also in the release notes https://github.com/lando/lando/releases/tag/v3.1.4

**Comment by sbubaron** [comment]
having the same issue despite lando 3.1.4 claiming to have added support for 1.29.1

installed Lando for first time today, followed vanilla drupal 7 recipe, setup database, successfully added content. performed a lando stop and start content was saved. rebooted system, when I went to start lando again got this error and the database seemed to disappear, had to reinstall drupal again

using Ubuntu 18 WSL2

Docker Desktop on Host (Windows 10 machine) is configured for Integration with default WSL distro and for Ubuntu 18.04 specifically (I think checking that caused the issue) but not sure. 

Docker Desktop on Windows host is v 3.3.0 (62916)
Engine is v20.10.5
Compose is v 1.29.0

docker --version from Ubuntu WSL says v20.10.6 build 370c289
docker-compose --version says 1.29.0 build 07737305

**Comment by mmodler** [comment]
We also see the message

**Comment by RishiKulshreshtha** [comment]
Getting the same message.


```
 ■ Using an unsupported version of DOCKER COMPOSE
   You have version 1.29.1 but Lando wants something in the 1.23.0 - 1.29.0 range.
   If you have purposefully installed an unsupported version and know what you are doing
   you can probably ignore this warning. If not we recommend you use a supported version
   as this ensures we can provide the best support and stability.
   https://docs.docker.com/compose/install/#install-compose-on-linux-systems
```

```
$ lando version
v3.1.4
```

```
$ docker–compose up

Command 'docker–compose' not found, did you mean:

  command 'docker-compose' from deb docker-compose (1.25.0-1)

Try: sudo apt install <deb name>
```

**Comment by mortona42** [comment]
I followed the instructions to downgrade my docker-compose to 1.29.0. `docker-compose version` gives 1.29.0, but lando still shows this warning with the version as 1.29.1.

**Comment by gitressa** [comment]
Using Ubuntu 20.04 and also getting this warning.

**Comment by dev286** [comment]
Just got this on Debian after installing 10.29.0.

**Comment by delacosta456** [comment]
hi i am having this issue too!
 please is there any solution

**Comment by pirog** [comment]
Safe to ignore. Will be fixed in next release.

**Comment by delacosta456** [comment]
hi
thanks and this has been fixed in 3.3
